### DISCUSSION TITLE
Sending data to centralized loki

### DISCUSSION THREAD

**Discussion by monomac** [netobserv/network-observability-operator#371] [General]
Hi guys,

is it possible or feasible to collect netobserv data from several clusters into one centralized loki? Let's say it's all installed via operators and I'm able to collect openshift-logging data  this way already. However, it seems to me netobserv is not being developed with such a possibility in mind.

Thank you

**Comment by jotak** [comment]
Hi @monomac ,

It's possible to have several instances of netobserv sending data to a centralized loki, yes. The address of the loki server is configurable, so you should be able to do that.
However there is some ongoing work to improve the experience on multi-clusters. For instance, all data received in Loki would currently be mixed up without the possibility to distinguish a cluster from another. Also, cross-cluster flows will just appear as flows to unknown / external ips.
So the current work is 1. to add cluster information in flows; 2. find solutions to better identify cross-cluster flows.
We use this repo for PoC in that area: https://github.com/netobserv/multi-cloud-research

cc @eranra @KalmanMeth @ronensc
